loadtest/control/gencontroller/actions.go

```diff
@@ -189,11 +189,23 @@ func (c *GenController) createPost(u user.User) control.UserActionResponse {
 	minWordCount := 1
 	wordCount := rand.Intn(avgWordCount*2-minWordCount*2) + minWordCount
 
-	postId, err := u.CreatePost(&model.Post{
+	post := &model.Post{
 		Message:   control.GenerateRandomSentences(wordCount) + channelMention,
 		ChannelId: channel.Id,
 		CreateAt:  time.Now().Unix() * 1000,
-	})
+	}
+
+	// 5% of the times post will have urgent priority.
```

Comment:
Where is this number coming from? Have we ran some queries on community to check this?